geomesa-accumulo/geomesa-accumulo-datastore/src/test/scala/org/locationtech/geomesa/accumulo/process/JoinProcessTest.scala

```diff
@@ -0,0 +1,65 @@
+/***********************************************************************
+* Copyright (c) 2013-2016 Commonwealth Computer Research, Inc.
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Apache License, Version 2.0
+* which accompanies this distribution and is available at
+* http://www.opensource.org/licenses/apache2.0.php.
+*************************************************************************/
+
+package org.locationtech.geomesa.accumulo.process
+
+import org.geotools.filter.text.ecql.ECQL
+import org.junit.runner.RunWith
+import org.locationtech.geomesa.accumulo.TestWithMultipleSfts
+import org.locationtech.geomesa.accumulo.process.unique.UniqueProcess
+import org.locationtech.geomesa.accumulo.util.SelfClosingIterator
+import org.locationtech.geomesa.features.ScalaSimpleFeature
+import org.specs2.mutable.Specification
+import org.specs2.runner.JUnitRunner
+
+@RunWith(classOf[JUnitRunner])
+class JoinProcessTest extends Specification with TestWithMultipleSfts {
+
+  sequential
+
+  val sft1 = createNewSchema("pk:String,age:Int,weight:Int,dtg:Date,*geom:Point:srid=4326")
+  val sft2 = createNewSchema("pk:String:index=true,dtg:Date,*geom:Point:srid=4326")
```

Comment:
[picky] Wouldn't the field in ```sft2``` be called ```fk``` (as it's a foreign-key reference) instead of ```pk```?